Library/Formula/rtpbreak.rb

```diff
@@ -1,11 +1,11 @@
-require 'formula'
-
 class Rtpbreak < Formula
-  homepage 'http://www.dallachiesa.com/code/rtpbreak/doc/rtpbreak_en.html'
-  url 'http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz'
-  sha1 'd22e9c37cc28c2fc36475d221b4eb2cc2c5aafbb'
+  # Homepage and URL dead since at least Feb 2015
+  homepage "http://www.dallachiesa.com/code/rtpbreak/doc/rtpbreak_en.html"
+  url "http://dallachiesa.com/code/rtpbreak/rtpbreak-1.3a.tgz"
+  mirror "https://raw.githubusercontent.com/DomT4/LibreMirror/master/Rtpbreak/rtpbreak-1.3a.tgz"
```

Comment:
Found a working link on a reasonably well-known security tools website, downloaded the ` .tgz `, checksummed it to check it matched the expected contents, and then stuck it on my mirror when it came back as a match.